src/main.ts

```diff
@@ -1106,6 +1129,176 @@ function generateServiceClientImpl(
   return client;
 }
 
+function generateNestjsService(
```

Comment:
Maybe `generateNestjsControllerInterface`? Or maybe `generateNestjsServiceController`?

Reply:
Agree `generateNestjsServiceController` is a better name for this.